components.py

```diff
@@ -15,23 +13,21 @@ def Number(id, default, min, max):
     ----------
     id : str
         ID used for Dash callbacks.
-    default : int
+    default : float
         Default value that is displayed in the input box when user loads the page.
-    min : int
-        Minimum value the user can select from.
-    max : int
-        Maximum value the user can select from.
+    min : float, optional
+        Minimum value the user can select from. Default is 0.
+    max : float, optional
+        Maximum value the user can select from. Defautl is 5.
+
     Returns
     -------
     dbc.Input : component
         dbc Input component with numerical inputs.
-    Notes
-    -----
-    Examples
-    --------
     """
-    return dbc.Input(id=id, type="number", min=min, max=max, size="sm",
-                     className="m-1 d-inline-block w-auto",debounce=True,value=str(default))
+    return [dbc.Input(id=id, type="number", min=min, max=max, size="sm",
+                     className="m-1 d-inline-block w-auto", debounce=True, value=str(default))]
```

Comment:
I feel like assigning these presentation-related CSS classes here is mixing content and presentation, but I really have no idea how bootstrap is meant to be used. @kgraaf any thoughts?

Reply:
Yeah it probably is, but at the same time we've already entangled presentation and content too much as it is in this manner so I don't know how much this matters.